### DISCUSSION TITLE
New wiki page???

### DISCUSSION THREAD

**Discussion by huwdp** [lightspark/lightspark#1074] [General]
@dbluelle, I see someone has changed the wiki but I have no idea who did this or how they did it.

![image](https://github.com/lightspark/lightspark/assets/364976/eeda8e59-7718-40a0-a0ca-cd5cec87a2b4)
![image](https://github.com/lightspark/lightspark/assets/364976/300d64e3-0364-4c48-a3bb-b4a2a2832db6)

**Comment by dbluelle** [comment]
Hmm, seems to be gone now, maybe that was some bug introduced by the github staff...
